Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml

```diff
@@ -800,6 +800,22 @@ tasks:
                     right:
                       value:
                         simple: docm
+                - - operator: isNotEqualString
```

Comment:
Please add the the same filter to the task that extracts. If a word + OFT files will be present, then it will still extract both.
Also add ignore-case.

Reply:
Done